Unix/scripts/config_keytab_update.sh

```diff
@@ -69,11 +78,13 @@ do
     case "$1" in
 	--configure)
         configure
+        cleanTempFile
 	    shift 1
 	    ;;
 
 	--unconfigure)
         unconfigure
+        cleanTempFile
```

Comment:
Probably better to call cleanTempFile after this do loop, so it will cover the error case also.

Reply:
great! will fix it, thanks.